leshan-integration-tests/src/test/java/org/eclipse/leshan/integration/tests/QueueModeIntegrationTestHelper.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ * Copyright (c) 2016 Bosch Software Innovations GmbH and others.
```

Comment:
In this PR, this is new interfaces. (but as they have the same name as previous classes, so git diff show this like a modification)
(And the change is done a separate commit to facilitate review)
Do not hesitate to comment my PR, if you see something wrong :smile: